### DISCUSSION TITLE
just want to share my compliments on how this looks, great job !

### DISCUSSION THREAD

**Discussion by Vinisz** [zwave-js/node-red-contrib-zwave-js#133] [Ideas]
just want to share my compliments on how this looks, great job !

**Comment by crxporter** [comment]
Hello and welcome to the party!

Marcus has done a great job putting things together. He puts up well with our constant "how do we do that" and "why can't it do this" and on and on.

And from my part, I've been working on the help, readme, and "getting started" docs lately. I hope you find them useful! If you are confused about any part of it, don't hesitate to ask questions. I'll be getting more of the wiki soon - it needs more screenshots!

We are glad you're here! Thank's for reaching out. There's a discord board where we all hang out daily if you want to stop by sometime - that's [here](https://discord.gg/uvYac5u) just find the `#z-wave` channel.

**Comment by marcus-j-davies** [comment]
@Vinisz,

Any questions on this NR Module specifically, or the driver its based on (zwave-js) - please, don't hold back. 👍 
I would of course, play with a few devices before commiting to a full switch - given the size of the network you are dealing with.
